ext/nnstreamer/tensor_source/tensor_src_tizensensor.c

```diff
@@ -881,6 +884,13 @@ gst_tensor_src_tizensensor_start (GstBaseSrc * src)
     retval = FALSE;
     goto exit;
   }
+
+  /** @todo TBD. Let's assume each frame has a fixed size */
+  gst_base_src_set_dynamic_size (src, FALSE);
```

Comment:
It is OK to call set_synamic_size AFTER listener_start?

Reply:
blocksize is fixed value. I think this is ok to set the block size before calling gst_base_src_start_complete().

Reply:
this line is moved to _init() function.